src/org/daisy/dotify/hyphenator/impl/hyphenation-catalog.xml

```diff
@@ -96,7 +96,9 @@
 
     <entry key="ro">resource-files/ro.xml</entry>
 
+<!--
```

Comment:
Please add a comment that explains why you've made this change

Reply:
Agree.

Reply:
Comment added.